modules/webgl/src/classes/texture.js

```diff
@@ -163,7 +161,8 @@ export default class Texture extends Resource {
       dataFormat,
       border,
       mipmaps,
-      parameters: pixelStore
+      parameters: pixelStore,
+      compressed
     });
 
     if (mipmaps) {
```

Comment:
If compressed we don't want to generate mipmaps, so you could check for that here too, so that we don't need to specify `mipmaps: false`.